tasks/module_4/task_6.py

```diff
@@ -0,0 +1,25 @@
+"""Module4/Task6: Very sophisticated password breaker."""
+from PyPDF2 import PdfFileReader
+
+
+# Vars
+DICTIONARY_PATH = 'resources/task_6/dictionary.txt'
+PDF_PATH = 'resources/task_6/encrypted.pdf'
+
+
+def decrypt_pdf_password():
+    """Decrypt password for encrypted pdf from file of passwords."""
+    password = None
+
+    with open(DICTIONARY_PATH, 'r') as eng_dict, open(PDF_PATH, 'rb') as enc_pdf:
+        my_pdf = PdfFileReader(enc_pdf)
+        for word in eng_dict.read().splitlines():
```

Comment:
File object in Python is iterable, so you can also iterate over lines like this:
```python
for line in eng_dict:
    ...
```
The difference is that your solution first load whole file content into memory as a string and then split it into a list of lines. In case of big files it may take a lot of RAM.